### DISCUSSION TITLE
new to Github-wiki: can i add images to the github-Wiki-Site:  is this doable?

### DISCUSSION THREAD

**Discussion by fsj-digital** [community/community#53418] [New to GitHub]
hello dear Friends

i am currently reading through the docs - such as those here:

Adding or editing wiki pages https://docs.github.com/en/communities/documenting-your-project-with-wikis/adding-or-editing-wiki-pages

**question:**

can i add images to the wikipages!?

can i do this - and add som documents such as pdf

did you ever add some kind of forms to the wikipage!?

love to hear from you regards

**Comment by LKreutzer** [comment]
Yes you can add images to Wiki pages. When you are in [edit mode](https://docs.github.com/assets/cb-210259/mw-1000/images/help/wiki/wiki-edit-mode-dropdown.webp) you can click the icon with the "landscape", next to weblink, and you can insert an image URL there. See also [these instructions](https://docs.github.com/en/communities/documenting-your-project-with-wikis/editing-wiki-content#adding-images).

**Reply by LKreutzer** [reply]
Could you please mark my answer as correct if this helped you?

**Reply by fsj-digital** [reply]
hello dear LKreuzer many thanks for the quick reply - awesome. 

and according to this page
https://docs.github.com/en/communities/documenting-your-project-with-wikis/changing-access-permissions-for-wikis

i am able to make various changes to the permissions so that others  (than contributors to the repo) are allowed to add and contribute to the wiki . that is just awesome. 


**frankly**: with the reworked - Permissions on the wiki-page: 
i guess that every body can add and contribute to the wiki - every body that has got a github-account. 


regards

**Reply by LKreutzer** [reply]
Yep that is correct! The Wikis are pretty useful for open-source projects!

Could you please mark my initial comment as the answer of this thread - e.g. as shown here:
![Screenshot from 2023-04-21 17-09-01](https://user-images.githubusercontent.com/34488763/233671456-04d6199b-26cb-40b4-824e-8ee0cb28ff0a.png)
